tests/phpstan-baseline.neon

```diff
@@ -0,0 +1,109 @@
+parameters:
```

Comment:
To avoid making changes outside of adding the phpstan tooling to this pull request, we generate a baseline of "known issues" which can be fixed in follow-up pull requests.